amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/RetrievalConfig.java

```diff
@@ -119,9 +119,7 @@ public class RetrievalConfig {
 
     public RetrievalConfig(@NonNull KinesisAsyncClient kinesisAsyncClient, @NonNull String streamName,
                            @NonNull String applicationName) {
-        this(kinesisAsyncClient,
-                new SingleStreamTracker(streamName, kinesisAsyncClient.serviceClientConfiguration().region()),
-                applicationName);
+        this(kinesisAsyncClient, new SingleStreamTracker(streamName), applicationName);
```

Comment:
Added `RetrievalConfig(KinesisAsyncClient, Arn, String)` constructor. Confirmed with @lucienlu-aws that `creationEpoch` would not be an issue because this constructor can only be used for single-stream mode.